runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -9554,9 +9571,7 @@ class INTERPRETER_CLASS
 
 		/* Pop memberNameObject from the stack. */
 		j9object_t memberNameObject = *(j9object_t *)_sp++;
-		if (J9_UNEXPECTED(NULL == memberNameObject)) {
-			goto throw_npe;
-		}
+		if (nullCheckJ9Obj(memberNameObject, fromJIT, REGISTER_ARGS, true) == THROW_NPE) return THROW_NPE;
```

Comment:
These lines do not follow the coding standard - constant comparisons are (const == expr), all if/while/etc must use braces and multiple returns from a function are strongly discouraged. These changes are unnecessary to address the actual issue, so I would prefer they be removed.